app/models/player.rb

```diff
@@ -0,0 +1,2 @@
+class Player < ApplicationRecord
+end
```

Comment:
@kkkoech it's good practice to add model validations and not rely completely on the database constraints. the `players` database table is created not to allow null values for `first_name` and `last_name` so in this case you could have model validations (and appropriate tests:

`validates :first_name, presence: true`
`validates :last_name, presence: true`

This way, you can check for valid models (using the `.valid?` method) without attempting to save to the database (which will raise an exception if the record cannot be saved).